Marlin/src/module/temperature.cpp

```diff
@@ -858,18 +867,44 @@ volatile bool Temperature::raw_temps_ready = false;
 
       if (ELAPSED(ms, next_report_ms)) {
         next_report_ms += 1000UL;
-        SERIAL_ECHOLNPGM("Temperature ", current_temp);
+
+        print_heater_states(active_extruder);
```

Comment:
Note that if host temperature auto-reporting is enabled, this will produce some redundant reports. It's unclear whether users or hosts find extra value in this "Temperature 123" report being distinct from the standard temperature host-readable temperature report. Some hosts might be handling a running `M303` in some special way. If hosts are not actively monitoring `M303` then they may get more value from the standard temperature report string.